test/test_nn.py

```diff
@@ -3735,6 +3735,12 @@ def test_threshold_int(self):
         expected = torch.tensor([99, 99, 99, 99, 1, 2, 3])
         self.assertEqual(F.threshold(x, 0, 99), expected)
 
+    def test_threshold_bfloat16(self):
+        x = torch.randn(100)
+        expected = F.threshold(x, 0, 0).bfloat16().float()
+        res_bf16 = F.threshold(x.bfloat16(), 0, 0).float()
```

Comment:
Could you add a couple more tests on different threshold values?

Reply:
Negative and positive values, -inf and inf, and NaN please.